hw3/templates/register.html

```diff
@@ -0,0 +1,36 @@
+{% extends "base.html" %}
+{% block content %}
+<div align="center" class="col-10">
+<h1>Login</h1>
+<form method="POST" action="{{ url_for('register') }}">
+    {% with messages = get_flashed_messages(with_categories=true)
+%}
+{% if messages %}
+{% for category, message in messages %}
+<div class="alert alert-{{ category }}">
+{{ message }}
+</div>
+{% endfor %}
+{% endif %}
+{% endwith %}
+{{ form.csrf_token }}
+{% for field in form if field.name != 'csrf_token' %}
+<p>
+{{ field.label }}<br>
+{{ field }}
+{% if field.errors %}
+<ul class="alert alert-danger">
+{% for error in field.errors %}
+<li>{{ error }}</li>
+{% endfor %}
+</ul>
+{% endif %}
```

Comment:
Мне не кажется целесообразным использование условия `if`
```suggestion
<ul class="alert alert-danger">
{% for error in field.errors %}
<li>{{ error }}</li>
{% endfor %}
</ul>
```